app/celery/lookup_recipient_communication_permissions_task.py

```diff
@@ -72,21 +72,25 @@ def recipient_has_given_permission(
     except NoResultFound:
         current_app.logger.info('No communication item found for notification %s', notification_id)
 
-    # get default send flag when available
-    if communication_item is not None:
-        default_send_flag = communication_item.default_send_indicator
-    else:
+    if communication_item is None:
         # Calling va_profile without a communication item won't return anything.
         # Perform default behavior of sending the notification.
         return None
 
+    # get default send flag when available
+    default_send_flag = communication_item.default_send_indicator
+
     try:
         is_allowed = va_profile_client.get_is_communication_allowed(
-            identifier,
-            communication_item.va_profile_item_id if communication_item is not None else None,
-            notification_id,
-            notification_type
+            identifier, communication_item.va_profile_item_id, notification_id, notification_type
         )
+
+        current_app.logger.info('Value of permission for item %s for recipient %s for notification %s: %s',
+                                communication_item.va_profile_item_id if communication_item else None,
```

Comment:
You don't need to test "communication_item".  You already know it isn't None because you tested above.